### PR TITLE
Even more signals support from my LSNP PR

### PR DESCRIPTION
**Pull request by mbavaro**
Brings in the WIP https://github.com/swift-nav/libswiftnav-private/pull/1593

### CODE REVIEW

File: piksi_tools/console/utils.py [hunk 1]
```diff
@@ -32,71 +32,132 @@
...
+CODE_BDS3_B1CX = 46
+CODE_BDS3_B5I = 47
+CODE_BDS3_B5Q = 48
+CODE_BDS3_B5X = 49
+CODE_BDS3_B7I = 50
+CODE_BDS3_B7Q = 51
+CODE_BDS3_B7X = 52
+CODE_BDS3_B3I = 53
+CODE_BDS3_B3Q = 54
+CODE_BDS3_B3X = 55
```
Comment (ljbade) [inline piksi_tools/console/utils.py]: we will have to keep an eye out for a future RINEX spec in case they manage to pick different codes...

**Comment by mbavaro** [comment]
OK, I'll restore and fix the tests around signal and ask Paul to profile. If it looks good I'll merge.

File: piksi_tools/console/utils.py (hunk 1 above)
Comment (mbavaro) [reply piksi_tools/console/utils.py]: oh yes, these are hit-and-miss right now...

**Comment by mbavaro** [comment]
waiting to merge https://github.com/swift-nav/libswiftnav-private/pull/1593 and then I'll merge this one as well..

**Pull request merged**